rita_lab/src/panels/tabs/debug/central_panel.rs

```diff
@@ -29,29 +29,20 @@ pub fn show(
 }
 
 fn draw_triangles(plot_ui: &mut PlotUi, triangulation_data: &mut TriangulationData) {
-    for i in 0..triangulation_data.triangulation.tds().num_tris() {
-        let tri = triangulation_data.triangulation.tds().get_tri(i).unwrap();
-        let [n0, n1, n2] = tri.nodes();
-
-        if !tri.is_conceptual() {
-            let v0 = triangulation_data.vertices[n0.idx().unwrap()];
-            let v1 = triangulation_data.vertices[n1.idx().unwrap()];
-            let v2 = triangulation_data.vertices[n2.idx().unwrap()];
-
-            plot_ui.polygon(
-                Polygon::new(vec![v0, v1, v2])
-                    .name(format!("Triangle {}", i))
-                    .fill_color(Color32::from_rgba_premultiplied(46, 128, 115, 2))
-                    .width(1.0),
-            );
-        }
+    for (i, [a, b, c]) in triangulation_data.triangulation.tris().into_iter().enumerate() {
+        plot_ui.polygon(
+            // todo use borrowed series
+            Polygon::new(format!("Triangle {i}"), vec![a, b, c])
+                .fill_color(Color32::from_rgba_premultiplied(46, 128, 115, 2))
+                .width(1.0),
+        );
```

Comment:
for [a, b, c] in triangulation_data.triangulation.tris() {
        plot_ui.polygon(
            Polygon::new(vec![a, b, c])
                .stroke(Stroke::new(1.0, TRI_GREEN))
                .width(1.0),
        );
    }
```suggestion
    for [a, b, c] in triangulation_data.triangulation.tris() {
        plot_ui.polygon(
            // todo use borrowed series
            Polygon::new("", vec![a, b, c])
                .stroke(Stroke::new(1.0, TRI_GREEN))
                .width(1.0),
        );
```

Reply:
Is it not useful to have control over each triangle in debug mode?

Reply:
yeah i guess you're right, still at least lat use he more descriptive `.stroke()` method.